web/cobrands/hart/hart.scss

```diff
@@ -337,13 +337,18 @@ body.frontpage .table-cell .content {
 
     li {
       margin: 0 2px;
+
+      @media only screen and (max-width:48em) {
+        float: left;
+        margin-right: 5px;
```

Comment:
Took me a while to remember myself, but without floating the order appears incorrectly, and the spacing is uneven:

## Without

![without](https://cloud.githubusercontent.com/assets/109774/25531777/ca46e950-2c22-11e7-81a9-707352a3f665.png)

## With

![with](https://cloud.githubusercontent.com/assets/109774/25531802/d98acc88-2c22-11e7-973c-f1fc66df7739.png)